google/cloud/bigquery/client.py

```diff
@@ -75,6 +75,19 @@
 from google.cloud.bigquery.table import TableReference
 from google.cloud.bigquery.table import RowIterator
 
+# Types needed only for Type Hints
+import datetime
+import pandas
```

Comment:
Is there any way we can avoid this import? I would like to keep pandas as an optional dependency.

I recall sometimes strings can be used for type hints.

Reply:
I have tried, but lint was failing so removed pandas type hints